packages/cashc/src/generation/GenerateTargetTraversal.ts

```diff
@@ -354,7 +354,18 @@ export default class GenerateTargetTraversal extends AstTraversal {
       this.emit(hexToBin('87'));
       this.emit(Op.OP_CAT);
       this.popFromStack(2);
-    } else if (node.identifier.name === Class.LOCKING_BYTECODE_NULLDATA) {
+    } else if (node.identifier.name === Class.LOCKING_BYTECODE_P2SH32) {
+      // OP_HASH256 OP_PUSH<20>
+      this.emit(hexToBin('aa14'));
```

Comment:
This probably needs to be OP_PUSH<32> since we're pushing a 32 byte script hash right?

Reply:
You're right of course! My bad!

changed it to
`// OP_HASH256 OP_PUSH<32>
 this.emit(hexToBin('aa20'));`